app/mailers/spool10203_submissions_report_mailer.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'reports/uploader'
+
+class Spool10203SubmissionsReportMailer < ApplicationMailer
+  REPORT_TEXT = '10203 spool submissions report'
+  RECIPIENTS = %w[
```

Comment:
Would the recipients be better suited for the settings.yml?

Reply:
I can move them there

 I was following what https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/mailers/spool_submissions_report_mailer.rb is doing

Reply:
Ok, the settings.yml file just seems more fitting, but i'll leave it up to you.